spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -259,17 +259,20 @@ public SparkContext createSparkContext() {
 
     for (Object k : intpProperty.keySet()) {
       String key = (String) k;
-      Object value = intpProperty.get(key);
-      logger.debug(String.format("SparkConf: key = [%s], value = [%s]", key, value));
-      conf.set(key, (String) value);
+      String val = toString(intpProperty.get(key));
+
+      if (!val.startsWith("spark.") || !val.trim().isEmpty()) {
```

Comment:
@jongyoul indeed! Thank you. 
How can we test this except extracting a new helper method and testing it separately?